### PR TITLE
doc: security: cve-2024-3077 left embargo

### PR DESCRIPTION
**Pull request by ceolin**
Disclose information about CVE-2024-3077

### CODE REVIEW

**Review by kartben — DISMISSED** [review]

**Comment by ceolin** [comment]
@kartben just fixed the compliance complaint

**Review by kartben — DISMISSED** [review]

File: doc/security/vulnerabilities.rst [hunk 1]
```diff
@@ -1699,3 +1699,17 @@ This has been fixed in main for v3.6.0
 
 - `PR 69170 fix for main
   <https://github.com/zephyrproject-rtos/zephyr/pull/69170>`_
+
+CVE-2024-3077
+-------------
+
+Bluetooth: integer underflow in gatt_find_info_rsp. An malicious BLE
```
Comment (aescolar) [inline doc/security/vulnerabilities.rst:1706]:
```suggestion
Bluetooth: Integer underflow in gatt_find_info_rsp. A malicious BLE
```

**Review by aescolar — COMMENTED** [review]
Thanks @ceolin, just 2 silly typos.

**Review by aescolar — APPROVED** [review]

**Comment by ceolin** [comment]
@nashif @kartben can we merge it ? This cve is already public.

**Review by kartben — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**